### PR TITLE
Fixed FileNotFoundError caused by removing file after git add

### PR DESCRIPTION
**Pull request by jadchaar**
Closes: https://github.com/asottile/seed-isort-config/issues/37

### CODE REVIEW

File: tests/seed_isort_config_test.py [hunk 1]
```diff
@@ -286,3 +286,24 @@ def test_returns_zero_no_changes_pyproject_toml(tmpdir):
...
+        tmpdir.join('f.py').write('import pre_commit\n')
+        tmpdir.join('g.py').write('import cfgv\n')
+        _make_git()
+
+        assert main(()) == 1
+
+        expected = '[settings]\nknown_third_party=cfgv,pre_commit\n'
+        assert tmpdir.join('.isort.cfg').read() == expected
+
+        os.remove(str(tmpdir.join('g.py')))
```
Comment (asottile) [inline tests/seed_isort_config_test.py]: I believe you can use `tmpdir.join('g.py').remove()`

File: tests/seed_isort_config_test.py [hunk 2]
```diff
@@ -286,3 +286,24 @@ def test_returns_zero_no_changes_pyproject_toml(tmpdir):
...
+    with tmpdir.as_cwd():
+        tmpdir.join('.isort.cfg').write('[settings]\nknown_third_party=\n')
+        tmpdir.join('f.py').write('import pre_commit\n')
+        tmpdir.join('g.py').write('import cfgv\n')
+        _make_git()
+
+        assert main(()) == 1
+
+        expected = '[settings]\nknown_third_party=cfgv,pre_commit\n'
+        assert tmpdir.join('.isort.cfg').read() == expected
```
Comment (asottile) [inline tests/seed_isort_config_test.py]: you can remove 299 - 302, no need to duplicate the test above -- you really just want to test the specific failure mode here

File: tests/seed_isort_config_test.py [hunk 3]
```diff
@@ -286,3 +286,24 @@ def test_returns_zero_no_changes_pyproject_toml(tmpdir):
         assert main(()) == 0
 
         assert cfg.read() == '[settings]\nknown_third_party=["cfgv"]\n'
+
+
+# regression test for issue #37
```
Comment (asottile) [inline tests/seed_isort_config_test.py]: I'd put this as a docstring instead of a comment above the function

**Comment by jadchaar** [comment]
Thanks for the feedback. I have addressed the changes in the latest commit.

**Pull request merged**